apps/test/unit/code-studio/initSigninStateTest.js

```diff
@@ -22,34 +23,28 @@ describe('initSigninStateTest', () => {
       headerDiv.appendChild(name);
     }
 
-    const cookieName = '__testcookie__';
+    before(() => {
+      window.dashboard.rack_env = 'unit_test';
+      cookieName = environmentSpecificCookieName('_shortName');
+    });
+
     beforeEach(() => {
       cookies.remove(cookieName);
-      stashedCookieKey = window.userNameCookieKey;
-      delete window.userNameCookieKey;
     });
 
     afterEach(() => {
       if (headerDiv) {
         document.body.removeChild(headerDiv);
         headerDiv = null;
       }
-      window.userNameCookieKey = stashedCookieKey;
-    });
-
-    it('does not work if userNameCookieKey is not set', () => {
-      assert.strictEqual(getUserSignedInFromCookieAndDom(), undefined);
     });
 
     it('returns true if cookie is defined', () => {
-      window.userNameCookieKey = cookieName;
       cookies.set(cookieName, 'CoolUser');
       assert.strictEqual(getUserSignedInFromCookieAndDom(), true);
     });
 
     it('returns true if cookie is not defined but DOM contains id', () => {
-      window.userNameCookieKey = cookieName;
-
       // Make sure this DOM didn't leak in from some other test
       assert.equal(document.querySelector('.header_button.header_user.user_menu .user_name'), null);
```

Comment:
Looking at this again, I think I can actually get rid of the whole looking at the DOM portion of `getUserSignedInFromCookieAndDom`, but that's out of scope for this PR.